RealmModelGenerator/MainVCExtension.swift

```diff
@@ -8,7 +8,11 @@
 
 import Cocoa
 
+// --MARK-- temporary workaround
+fileprivate let VERSIONVC: NSStoryboard.SceneIdentifier = NSStoryboard.SceneIdentifier(rawValue: "VersionVC")
```

Comment:
static `let` are valid inside of extensions.
You could also declare at usage site. keeps the code "localized"

Reply:
SceneIdentifier should be in seprate file (and other ids)